src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Color {
+    RED,
+    BLUE,
+    BLACK,
+    WHITE,
```

Comment:
Consider using the `name()` method for getting the String representation of enum constants instead of `toString()` as it is safer and will not be affected by any overrides of the `toString()` method.